docs/develop/json-manifest-overview.md

```diff
@@ -1,40 +1,41 @@
 ---
-title: Teams manifest for Office Add-ins (preview)
-description: Get an overview of the preview JSON manifest.
+title:  Unified Microsoft 365 manifest (preview)
+description: Get an overview of the preview unified Microsoft 365 manifest.
```

Comment:
```suggestion
description: Get an overview of the preview JSON-formatted unified Microsoft 365 manifest.
```
Just a suggestion for your consideration. My thinking is that it may help with discovery if users search for "JSON manifest".